ocs_ci/ocs/ui/block_pool.py

```diff
@@ -173,3 +181,83 @@ def reach_pool_limit(self, replica, compression):
                     raise PoolStateIsUnknow(
                         f"pool {pool_name} is in unexpected state {pool_state}"
                     )
+
+    def pool_raw_capacity_loaded(self, pool_name):
+        """
+        Takes pool name and returns True if the raw capacity of the block pool is loaded
+        or returns False if the capacity is not loaded.
```

Comment:
please add 1 empty line in between docstring sections and apply in all functions.

```python
        """
        Takes pool name and returns True if the raw capacity of the block pool is loaded
        or returns False if the capacity is not loaded.

        Args:
            pool_name (str): The name of the pool to be deleted

        Returns:
            bool: True if raw capacity of the blockpool is loaded, otherwise False

        """
```

Reply:
Okay will update it

Reply:
> As per the description, the PR only runs up to OCP 4.9 which isn't the case in my opinion. Shouldn't it run on OCP/ODF higher than 4.9?

As mentioned in this description this PR is reopened PR from a closed one where it has passed tests for a higher version. Old PR: https://github.com/red-hat-storage/ocs-ci/pull/6169